src/components/range/range.ts

```diff
@@ -96,9 +97,6 @@ export default class SlRange extends LitElement {
     super.connectedCallback();
     this.resizeObserver = new ResizeObserver(() => this.syncRange());
 
-    if (!this.value) {
-      this.value = this.min;
-    }
```

Comment:
@claviska Not sure if this was intended to be like this, but I had to remove it in order to be able to use a negative min value and a starting value of zero.

Reply:
Everything seems fine, so I'm not concerned about it.